app/models/outbound_webhook.rb

```diff
@@ -20,17 +20,25 @@ def deliver(deploy, output)
     prefix = "Webhook notification:"
     output.puts "#{prefix} sending to #{url} ..."
 
-    response = connection.post url do |req|
-      req.headers['Content-Type'] = 'application/json'
-      req.body = self.class.deploy_as_json(deploy).to_json
-    end
+    error_message =
+      begin
+        response = connection.post url do |req|
+          req.headers['Content-Type'] = 'application/json'
+          req.body = self.class.deploy_as_json(deploy).to_json
+        end
+
+        if response.success?
+          output.puts "#{prefix} succeeded"
```

Comment:
This is expecting the webhook to do all of it's logic end to end before reporting success or failure. Would prefer something like the image builds where we can report back a status or we can provide a url to poll. Things on our end happen asynchronously and we can't be sure that the pod that receives the complete (or error) message is the same pod with the open connection.

Reply:
There will be another PR that allows the receiver to provide a url to poll.